components/core/src/clp/clo/CommandLineArguments.cpp

```diff
@@ -8,10 +8,8 @@
 #include <boost/program_options.hpp>
 
 #include "../cli_utils.hpp"
-#include "../reducer/types.hpp"
```

Comment:
I guess you removed this because CLion said it's unused? Our guidelines specify that we should include headers for things directly used in the file and there's a new clang-tidy warning for it: https://clang.llvm.org/extra/clang-tidy/checks/misc/include-cleaner.html

Reply:
Right. Do we have `.clang-tidy` file to check such warnings?

Reply:
I think the one we've been using should trigger it, but I'll check again later.

Reply:
I only find `.clang-tidy` in submodules

Reply:
It's the one we were using when open-sourcing clp-s. Anyway, we can figure this out offline.